### PR TITLE
Subtask/as 88 improved schema

### PR DESCRIPTION
**Pull request by swdpcomputing**
## Ticket Number

AS-88

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/appeal-reply-service-api/src/__tests__/reply.spec.js [hunk 1]
```diff
@@ -58,7 +55,8 @@ describe('Replies API', () => {
   test('GET /api/v1/reply/{id} - It responds with an existing reply', async () => {
     const reply = await createReply();
     const response = await request(app).get(`${endpoint}/${reply.id}`);
-    expect(response.body).toEqual(reply);
+    expect(JSON.stringify(response.body)).toBe(JSON.stringify(reply));
+    console.log(response.body);
```
Comment (davehlewis) [inline packages/appeal-reply-service-api/src/__tests__/reply.spec.js:59]: this needs to be removed

File: packages/appeal-reply-service-api/src/controllers/reply.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 const uuid = require('uuid');
 const logger = require('../lib/logger');
 const mongodb = require('../db/db');
-const { blankModel } = require('../models/blankModel');
+// const { blankReply } = require('../models/blankReply');
```
Comment (davehlewis) [inline packages/appeal-reply-service-api/src/controllers/reply.js:4]: any reason for this comment to be here?

**Review by davehlewis — CHANGES_REQUESTED** [review]

File: packages/appeal-reply-service-api/src/__tests__/reply.spec.js (hunk 1 above)
Comment (swdpcomputing) [reply packages/appeal-reply-service-api/src/__tests__/reply.spec.js:59]: Done.

File: packages/appeal-reply-service-api/src/controllers/reply.js (hunk 2 above)
Comment (swdpcomputing) [reply packages/appeal-reply-service-api/src/controllers/reply.js:4]: Removed

**Review by davehlewis — APPROVED** [review]

**Pull request merged**